madhanmellanox/sonic-buildimage
# Accomadating lower case ACL rules for Control Plane ACLs

**- Why I did it**
To make Control plane ACLs handle case insensitive ACL rules. Currently, it handles only upper case ACL rules.

**- How I did it**
I set the rule prop dictionary to be all upper case before reading the rule props for processing.

**- How to verify it**
Configure ACL rules with any combination of case of ACL rules, either it be upper case or lower case. All rules should be created properly in the switch and working properly.

**- Which release branch to backport (provide reason below if selected)**



- [ x] 201911

**- Description for the changelog**

The change is a one line change in caclmgrd script.

**- A picture of a cute animal (not mandatory but encouraged)**
